qemu/deps/playbook.yml

```diff
@@ -110,3 +114,9 @@
       shell:
         echo "Run test case failed"
       failed_when: run_case_result is failed
+
+    - name: Handle cancelled test cases
+      fail:
+        msg: "Run tests in non-gating, handel cancelled test cases as failed"
```

Comment:
@qcheng-redhat  The msg need be updated too, It is handle cancelled test case according to different usage.

Reply:
Updated fail msg, please review.

Reply:
@qcheng-redhat  `Run tests in non-gating environment, ***`.   This is not sutiable.
`non-gating` is only one usage.  others usages also can/can't accept cancel.
We should explain the problem itself, not one usage.

Reply:
Updated fail msg. Please review.